aicsimageio/readers/tiff_glob_reader.py

```diff
@@ -114,7 +124,7 @@ def _is_supported_image(
     def __init__(
         self,
         glob_in: Union[types.PathLike, List[types.PathLike]],
-        indexer: Union[pd.DataFrame, Callable] = None,
+        indexer: Union[pd.DataFrame, Callable, str] = None,
```

Comment:
I think my only last comment is that indexer now no longer accepts `str` type.

Reply:
yup. fixed!